examples/contracts/cw20-base/src/orch.rs

```diff
@@ -0,0 +1,89 @@
+// This contains all code we needed to manually add to make it work well with cw-orch
+// In the future, hopefully some of this can me auto-generated. But let's get it to work now.
+use crate::contract::entry_points::{execute, instantiate, query};
+use crate::contract::sv::{ContractExecMsg, ContractQueryMsg, ContractSudoMsg, InstantiateMsg};
+use cw_orch::prelude::*;
+
+// Maybe uploadable can be autogenerated?
+// But this is fine to include in the client code
+
+#[cw_orch::interface(InstantiateMsg, ContractExecMsg, ContractQueryMsg, Empty)]
```

Comment:
We can try to auto-generate this, or just leave this done by hand in the contracts that enable the "orch" feature